test_constraints.txt

```diff
@@ -8,5 +8,8 @@
 # constraint if it is installed either directly or transitively by the
 # dependencies.
 
-# TODO(b/220800338): Workaround for markupsafe.
```

Comment:
Could you revive this markupsafe constraint?
It was changed in https://github.com/tensorflow/tfx/commit/b021a486c9363147eca35e97742b555a5ad1cce4 but I don't want to include it in 1.8.x if possible.

Reply:
Sure jiyong. Can you please review again. Thanks.